plugin/cli.go

```diff
@@ -63,11 +64,11 @@ func getSignedJwt(role string, m map[string]string) (string, error) {
 		return "", fmt.Errorf("could not convert JWT payload to JSON string: %v", err)
 	}
 
-	jwtReq := &iam.SignJwtRequest{
+	jwtReq := &iamcredentials.SignJwtRequest{
 		Payload: string(payloadBytes),
 	}
 
-	iamClient, err := iam.New(httpClient)
+	iamClient, err := iamcredentials.New(httpClient)
 	if err != nil {
 		return "", fmt.Errorf("could not create IAM client: %v", err)
 	}
```

Comment:
On line 76: Wondering if `resourceName := fmt.Sprintf("projects/%s/serviceAccounts/%s", project, serviceAccount)`, should instead be `resourceName := fmt.Sprintf(gcputil.ServiceAccountCredentialsTemplate, serviceAccount)`? The migration doc seems to suggest the `-` character should always represent the project ID.

Reply:
Probably so, great catch! I updated that in 633d077